site/ko/alpha/tutorials/text/text_generation.ipynb

```diff
@@ -0,0 +1,1885 @@
+   "source": [
+    "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/r2/guide/effective_tf2.md)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃헙 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로 메일을 보내주시기 바랍니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "BwpJ5IffzRG6"
+   },
+   "source": [
+    "이 튜토리얼에서는 문자열 기반 순환 신경망(RNN, Recurrent Neural Network)을 사용하여 어떻게 텍스트를 생성하는지를 설명합니다.  우리는 Andrej Karpathy의 [순환 신경망의 비합리적 효과](http://karpathy.github.io/2015/05/21/rnn-effectiveness/)에서 셰익스피어의 작품을 데이터 셋으로 작업할 것입니다. 이 데이터(\"Shakespear\")에서 문자 시퀀스가 주어지면, 시퀀스(\"e\")의 다음 문자열을 예측하는 모델을 훈련합니다. 모델을 반복적인 호출은 더 긴 텍스트 시퀀스 생성이 가능합니다.\n",
+    "\n",
+    "Note: 이 노트북을 더 빠르게 실행하기 위해 GPU 가속을 활성화합니다. 코랩(Colab)에서: *Runtime > Change runtime type > Hardware acclerator > GPU 탭을 선택합니다. 로컬에서 실행하려면 TensorFlow 버전이 1.11 이상이어야 합니다.\n",
+    "\n",
+    "이 튜토리얼은 [tf.keras](https://www.tensorflow.org/programmers_guide/keras)와 [즉시 실행(eager execution)](https://www.tensorflow.org/programmers_guide/eager)을 활용하여 구현된 실행 가능한 코드가 포함되어 있습니다. 다음은 이 튜토리얼의 30번의 에포크(Epoch)로 훈련된 모델에서 \"Q\" 문자열로 시작될 때의 샘플 출력입니다.\n",
+    "\n",
+    "<pre>\n",
+    "QUEENE:\n",
+    "I had thought thou hadst a Roman; for the oracle,\n",
+    "Thus by All bids the man against the word,\n",
+    "Which are so weak of care, by old care done;\n",
+    "Your children were in your holy love,\n",
+    "And the precipitation through the bleeding throne.\n",
+    "\n",
+    "BISHOP OF ELY:\n",
+    "Marry, and will, my lord, to weep in such a one were prettiest;\n",
+    "Yet now I was adopted heir\n",
+    "Of the world's lamentable day,\n",
+    "To watch the next way with his father with his face?\n",
+    "\n",
+    "ESCALUS:\n",
+    "The cause why then we are all resolved more sons.\n",
+    "\n",
+    "VOLUMNIA:\n",
+    "O, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, no, it is no sin it should be dead,\n",
+    "And love and pale as any will to that word.\n",
+    "\n",
+    "QUEEN ELIZABETH:\n",
+    "But how long have I heard the soul for this world,\n",
+    "And show his hands of life be proved to stand.\n",
+    "\n",
+    "PETRUCHIO:\n",
+    "I say he look'd on, if I must be content\n",
+    "To stay him from the fatal of our country's bliss.\n",
+    "His lordship pluck'd from this sentence then for prey,\n",
+    "And then let us twain, being the moon,\n",
+    "were she such a case as fills m\n",
+    "</pre>\n",
+    "\n",
+    "문장 중 일부는 문법적으로 맞지만 대부분 자연스럽지 않습니다. 이 모델은 단어의 의미를 학습하지는 않았지만, 고려해야 할 점으로:\n",
+    "\n",
+    "* 모델은 문자열 기반입니다. 훈련이 시작되었을 때, 이 모델은 영어 단어의 철자법을 모르거나 그 단어가 심지어 텍스트 단위였습니다.\n",
+    "\n",
+    "* 출력의 구조는 일반적으로 데이터 세트와 비슷한 모든 대문자로 된 화자의 이름으로 시작하는 텍스트의 재생 블록과 유사합니다.\n",
```

Comment:
How about 연극 -> 극중 대본?
I think that the output is like 대본 more than 연극.
play script means 극중 대본 so I think it's better.

Reply:
Hi, @JKIsaacLee Thanks for translation and @cre8tor thanks for reviewing. :)
I suggest '대본', because they just use scripts.

'in all capital letters similar to the dataset' means all speaker name is capital.
So I suggest '출력의 구조는 대본과 닮았습니다. 즉, 텍스트 블록은 대개 화자의 이름으로 시작합니다. 이 이름들은 데이터셋에 모두 대문자로 씌여 있습니다.' :)